examples/Atlas/runDRCPracticeTerrain.m

```diff
@@ -4,7 +4,7 @@
 checkDependency('iris');
 checkDependency('mosek');
 task_number=2; % consider taking the task number as an input if/when we load the other tasks
-options.atlas_version = 3;
+options.atlas_version = 5;
```

Comment:
not required for this pull request, but it seems like v3 isn't going to work anymore (joint names changed, etc).  we should probably remove this option?